src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/EnvironmentTests.cs

```diff
@@ -579,11 +579,20 @@ public void TestCpuUsage()
         {
             if ((OperatingSystem.IsIOS() && !OperatingSystem.IsMacCatalyst()) || PlatformDetection.IstvOS || PlatformDetection.IsBrowser)
             {
-                // Environment should return 0 for all values
                 Environment.ProcessCpuUsage usage = Environment.CpuUsage;
-                Assert.Equal(TimeSpan.Zero, usage.UserTime);
-                Assert.Equal(TimeSpan.Zero, usage.PrivilegedTime);
-                Assert.Equal(TimeSpan.Zero, usage.TotalTime);
+
+                if (usage.UserTime == TimeSpan.Zero)
+                {
+                    // Environment should return 0 for all values
+                    Assert.Equal(TimeSpan.Zero, usage.PrivilegedTime);
+                    Assert.Equal(TimeSpan.Zero, usage.TotalTime);
+                }
+                else
+                {
+                    // Mobile platforms emulators may return non-zero values
```

Comment:
The comment says "may return non-zero values". If this is referring to PrivilegedTime, does that mean it could it be zero? Because in that case, the Assert.NotEqual would fail.

Reply:
The comment says `non-zero values` plural referring to all values returning in CpuUsage. if UserTime is not zero, PrivilegedTime shouldn't be zero either.